test/runtime/samples/empty-component-destroy/Empty.svelte

```diff
@@ -0,0 +1,8 @@
+
+<script>
+	import { onMount } from 'svelte';
+
+	onMount(() => {
+		return () => console.log('destroy');
+	});
```

Comment:
```suggestion
	import { onDestroy } from 'svelte';
	
	onDestroy(() => {
		console.log('destroy');
	});
```

Here we should check the Empty is destroyed.

Reply:
Already updated.